### PR TITLE
Le  1213

### PR DESCRIPTION
**Pull request by LuckyEu**
created the lass EntityEventsChain1Test

### CODE REVIEW

**Comment by LuckyEu** [comment]
created the lass EntityEventsChain1Test

**Comment by Stan575** [comment]
Good practice to create PR when some functionality is ready, tested and performs well. Till that it's fine to keep intermediate commits in your working branch

File: src/test/java/EntityEventsChain1Test.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+        WebDriver driver = getDriver();
+        WebDriverWait wait = new WebDriverWait(getDriver(), 1);
+        driver.get("https://ref.eteam.work");
+
+        ProjectUtils.login(driver, "user1@tester.com", "ah1QNmgkEO");
+
+        WebElement tab = driver.findElement(By.xpath("//*[@id=\"menu-list-parent\"]/ul/li[18]/a"));
+
+//        tab.click();
```
Comment (SergeiDemyanenko) [inline src/test/java/EntityEventsChain1Test.java:22]: Коментарии недопустимы в коде

**Comment by SergeiDemyanenko** [comment]
Что делает этот тест?

**Pull request closed without merging**